yrs/src/store.rs

```diff
@@ -132,6 +133,52 @@ impl Store {
         value.clone()
     }
 
+    pub(crate) fn find_index_clean_start(&mut self, client: &u64, clock: u32) -> Option<usize> {
```

Comment:
These two methods were just moved from `Transaction` to `Store` to make them more precise - Rust borrow checker likes precise field access.